### ISSUE TITLE
SimpleActionServer should expose dispose()

### ISSUE THREAD

**Issue by aez** [RobotWebTools/roslibjs#335]
There is currently no way to tear down a SimpleActionServer instance. A dispose() method seems like a reasonable thing to add to the API.

**Comment by Rayman** [comment]
Indeed, it should be relatively easy to implement. In the action client it's implemented here: https://github.com/RobotWebTools/roslibjs/blob/f0d6d6a6dcaadb8abacb47a5a82b126af79f84e8/src/actionlib/ActionClient.js#L135
